plugins/auth-backend/src/service/router.ts

```diff
@@ -53,8 +52,8 @@ export async function createRouter(
   });
 
   router.use(cookieParser());
-  router.use(bodyParser.urlencoded({ extended: false }));
-  router.use(bodyParser.json());
+  router.use(express.urlencoded({ extended: false }));
+  router.use(express.json());
```

Comment:
express has a dependency already to body-parser and does a passthrough - wanted to make this consistent across the codebase